src/components/DocumentationTopic.vue

```diff
@@ -293,6 +297,14 @@ export default {
       objcPath && swiftPath && isTargetIDE
     ),
     enhanceBackground: ({ symbolKind }) => (symbolKind ? (symbolKind === 'module') : true),
+    extraPadding: ({
+      roleHeading,
+      abstract,
+      sampleCodeDownload,
+      hasAvailability,
+    }) => (
+      (!!roleHeading + !!abstract + !!sampleCodeDownload + !!hasAvailability) <= 1
```

Comment:
I would have never in a million years thought about adding true/false values to sum up a number 🤣  Its genius and crazy at the same time.

OK so we dont apply padding, if we have 2 or more of those content items, correct?

Reply:
The rule is we don't apply extra padding if we have 3 or more items in the hero section. So we do apply extra padding if we have 1 or 2 items. However, a page will always have a title, which means we need extra padding when there are only 0 or 1 other items. (Hence the `<=1` )

Reply:
It would be good to add a comment explaining this.